sql/AddDeliveryEmp.sql

```diff
@@ -0,0 +1,11 @@
+# Use this SQL file to add new Delivery Employees to the database.
+# Change values in brackets
+#
+# In order to add multiple, copy line and change values
+# EG:
+# Insert into DeliveryEmployee(DEmpForename, DEmpSurname, DEmpBankNum, NationalInsuranceNum) VALUES
+# ('Forename','Surname','PhoneNum', 'NI Num'),
+# ('Forename','Surname','PhoneNum', 'NI Num');
+#
+Insert into DeliveryEmployee(DEmpForename, DEmpSurname, DEmpBankNum, NationalInsuranceNum) VALUES
+('Forename','Surname','PhoneNum', 'NI Num');
```

Comment:
What sort of person is Mr Forename Surname? :P Surely you need more data than just one entry? And 'NI Num' doesn't seem like a great example!